lib/node-helpers.js

```diff
@@ -31,21 +31,22 @@ function findFirstNamed(node, type, includeSelf = false) {
   }
 }
 
-function findAll(node, type, options) {
+function findAll(node, types, options) {
+  if (typeof(types) === 'string') types = [types]
   const results = []
   const nodeTypeToStopAt = options && options.stopAt
-  findAllHelper(node, type, results, nodeTypeToStopAt)
+  findAllHelper(node, types, results, nodeTypeToStopAt)
   return results
 }
 
-function findAllHelper(node, type, results, nodeTypeToStopAt) {
+function findAllHelper(node, types, results, nodeTypeToStopAt) {
   node.namedChildren.forEach(child => {
-    if (child.type === type) results.push(child)
+    if (types.some(type => child.type === type)) results.push(child)
```

Comment:
This could be written slightly shorter and faster as `types.includes(child.type)`

Reply:
Good point =]